Assets/01_Scripts/SpawnObject.cs

```diff
@@ -22,8 +25,9 @@ void Start()
     private IEnumerator Spawning()
     {
         while (true) {
-            
-            GameObject A = (GameObject)Instantiate(Platform);
+
+            randomPrefab = Random.Range(0, arrayCounter);
+            GameObject A = (GameObject)Instantiate(Prefabs[randomPrefab]);
```

Comment:
Could you rename `A` to something more meaningful? `A` is just too generic of a name

Reply:
I will